pallet-logion-loc/src/mock.rs

```diff
@@ -169,6 +169,18 @@ parameter_types! {
         reserve_percent: Percent::from_percent(0),
         treasury_percent: Percent::from_percent(100),
     };
+    pub const CollectionItemFeeDistributionKey: DistributionKey = DistributionKey {
+        stakers_percent: Percent::from_percent(25),
+        collators_percent: Percent::from_percent(25),
+        reserve_percent: Percent::from_percent(25),
+        treasury_percent: Percent::from_percent(25),
+    };
+    pub const TokensRecordFeeDistributionKey: DistributionKey = DistributionKey {
+        stakers_percent: Percent::from_percent(30),
+        collators_percent: Percent::from_percent(30),
+        reserve_percent: Percent::from_percent(30),
+        treasury_percent: Percent::from_percent(10),
```

Comment:
Same remark as above and same distribution key.

Reply:
Done as specified.